src/Docker.PowerShell/Cmdlets/GetContainerImage.cs

```diff
@@ -25,10 +37,17 @@ public class GetContainerImage : DkrCmdlet
         /// </summary>
         protected override async Task ProcessRecordAsync()
         {
-            foreach (var img in await DkrClient.Images.ListImagesAsync(
-                new ImagesListParameters() { All = All.ToBool() }))
+            var listParams = new ImagesListParameters() { All = (All || Id != null) };
+
+            foreach (var img in await DkrClient.Images.ListImagesAsync(listParams))
             {
-                WriteObject(img);
+                if (Id == null ||
+                    Id.Any(i => img.ID.Equals("sha256:" + i)) ||
```

Comment:
This can just be the `StartsWith` call right? There is no need to check for equality too.